### PR TITLE
Kaizen, Pleo and Vision-Box added.

### PR DESCRIPTION
**Pull request by fhbranco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Farfetch](https://www.farfetch.com) [:rocket:](https://farfetchgroupcareers.com/jobs) | Global technology platform for luxury fashion. | `Braga` `Lisboa` <br> `Porto` |
 | [Jumia](https://group.jumia.com) [:rocket:](https://group.jumia.com/careers) | Online Shopping for Electronics, Phones & Fashion in Africa. | `Porto` |
 | [Jumpseller](https://jumpseller.com) [:rocket:](https://jumpseller.com/jobs) | E-commerce platform to create online stores. | `Porto` |
+| [Kaizen Gaming](https://kaizengaming.com/home) [:rocket:](https://careers.kaizengaming.com/home) | Online betting and gambling operator. | `Lisboa` |
```
Comment (marmelo) [inline README.md:199]: I would remove the path to be future-proof.

```suggestion
| [Kaizen Gaming](https://kaizengaming.com) [:rocket:](https://careers.kaizengaming.com) | Online betting and gambling operator. | `Lisboa` |
```

File: README.md [hunk 2]
```diff
@@ -285,6 +286,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Kraken](https://www.kraken.com) [:rocket:](https://www.kraken.com/careers) | Kraken is a cryptocurrency exchange and bank. | `Remote` |
 | [LoanDolphin](https://loandolphin.com.au/) [:rocket:](https://loandolphin.breezy.hr/) | Home loan marketplace. | `Lisboa` `Remote` |
 | [Moonpay](https://www.moonpay.com) [:rocket:](https://www.moonpay.com/careers) | Solutions for payments, smart contracts, and digital asset management. | `Lisboa` `Remote` |
+| [Pleo](https://www.pleo.io/pt) [:rocket:](https://www.pleo.io/en/careers/jobs) | Centralise your business spending for complete financial efficiency and balance the books effortlessly. | `Lisboa` `Remote` |
```
Comment (marmelo) [inline README.md:289]: Same here.

```suggestion
| [Pleo](https://www.pleo.io) [:rocket:](https://www.pleo.io/en/careers/jobs) | Centralise your business spending for complete financial efficiency and balance the books effortlessly. | `Lisboa` `Remote` |
```

Comment (marmelo) [inline README.md:289]: The description is a bit long, could you please shorten it?

File: README.md [hunk 3]
```diff
@@ -421,4 +423,4 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Hostelworld](https://www.hostelworld.com/) [:rocket:](https://www.hostelworldgroup.com/careers) | World's leading hostel-focused online booking platform. | `Porto` |
 | [Ryanair](https://www.ryanair.com/pt/pt) [:rocket:](https://careers.ryanair.com/jobs/?ryanair-jobs-department=21233) | Ryanair is an Irish ultra low-cost carrier. | `Lisboa` `Porto`|
 | [TripAdvisor](https://www.tripadvisor.com) [:rocket:](https://careers.tripadvisor.com) | The largest "social travel website" in the world. | `Lisboa` |
-
+| [Vision-Box](https://www.vision-box.com/) [:rocket:](https://visionbox.teamtailor.com/jobs) | End-to-end solutions that combine a comprehensive software platform with seamless and contactless state-of-the-art equipment, making a travel journey feel effortless. | `Lisboa` |
```
Comment (marmelo) [inline README.md:426]: The description is a very long, could you please shorten it?

**Review by marmelo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fhbranco) [reply README.md]: Changed the description, hopefully is more aligned.

File: README.md (hunk 3 above)
Comment (fhbranco) [reply README.md]: Changed the description, hopefully is more aligned.

**Review by afonsojramos — APPROVED** [review]

**Review by marmelo — APPROVED** [review]

**Comment by marmelo** [comment]
Thank you @fhbranco!

**Pull request merged**